src/components/navigation/components/Header/Header.tsx

```diff
@@ -19,8 +19,8 @@ const b = block('header');
 const ICON_SIZE = 36;
 
 export interface HeaderProps {
-    logo: NavigationLogo;
-    data: HeaderData;
+    logo: NavigationLogoProps;
+    data: HeaderDataProps;
```

Comment:
usually we use `props` in naming for react components props. Try to use another naming